app.R

```diff
@@ -131,7 +107,7 @@ get_tides <- function(location) {
   
   df <- noaaoceans::query_coops_data(
     station_id = station_id,
-    start_date = format(lubridate::with_tz(Sys.time(), "America/New_York") %>% as.Date(), "%Y%m%d"),
+    start_date = format(lubridate::with_tz(Sys.time(), "America/New_York") %>% as.Date()-1, "%Y%m%d"),
```

Comment:
so the fix is to pull over three days? smart

Reply:
We'll see if it works correctly! Will have to check it later tonight when the next time is tomorrow